client/src/views/files/handler/editor/FileEditor.vue

```diff
@@ -140,38 +222,59 @@ async function openFileWithCryptpad(): Promise<void> {
   try {
     await cryptpadApi.open(cryptpadConfig);
     window.electronAPI.log('info', 'CryptPad editor initialized successfully');
-  } catch (error) {
-    throwError(`Failed to open CryptPad editor: ${error}`);
+  } catch (e: unknown) {
+    let title: Translatable = 'fileViewers.errors.titles.genericError';
+    const message: Translatable = 'fileViewers.errors.informationEditDownload';
+    const level: InformationLevel = InformationLevel.Info;
+
+    if (e instanceof CryptpadError) {
+      switch (e.code) {
+        case CryptpadErrorCode.NotInitialized:
+          title = 'fileViewers.errors.titles.genericError';
+          break;
+        case CryptpadErrorCode.DocumentTypeNotEnabled:
+          title = 'fileViewers.errors.titles.editionNotAvailable';
+      }
+    }
+
+    await informationManager.present(
+      new Information({
+        title,
+        message,
+        level,
+      }),
+      PresentationMode.Modal,
+    );
   }
 }
 
-function validateContentInfo(contentInfo: FileContentInfo | undefined): void {
+function validateContentInfo(contentInfo: FileContentInfo | undefined): string | undefined {
   if (!contentInfo) {
-    throw new Error('Content info is not available');
-  }
-
-  if (!contentInfo.extension || contentInfo.extension.trim() === '') {
-    throw new Error('File extension is missing or empty');
-  }
+    return (error.value = 'fileEditors.errors.contentInfoMissing');
+  } else {
+    if (!contentInfo.extension || contentInfo.extension.trim() === '') {
+      return (error.value = 'fileEditors.errors.fileExtensionMissing');
+    }
 
-  if (!contentInfo.fileName || contentInfo.fileName.trim() === '') {
-    throw new Error('File name is missing or empty');
-  }
+    if (!contentInfo.fileName || contentInfo.fileName.trim() === '') {
+      return (error.value = 'fileEditors.errors.fileNameMissing');
```

Comment:
Could have been cleaver, but no. This changes the prototype of the function, for no reason at all.
Sorry, but you have to write the full
```typescript
error.value = 'xxx';
return;
```